### PR TITLE
Feature/reroute

### PR DESCRIPTION
**Pull request by pnuts2**

### CODE REVIEW

File: Classes/model/CoreData/Alert.h [hunk 1]
```diff
@@ -31,6 +31,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, retain, nullable) NSDate * startTime;
 @property (nonatomic, retain, nullable) NSDate * endTime;
 @property (nonatomic, assign) BOOL toDelete;
+@property (nonatomic, retain, nullable) NSDictionary *action;
```
Comment (nighthawk) [inline Classes/model/CoreData/Alert.h]: What kind of dictionary is this? `<String, id>`?

File: Classes/model/CoreData/Alert.swift [hunk 2]
```diff
@@ -14,7 +14,11 @@ extension Alert {
     switch alertSeverity {
     case .info, .warning: return .warning
     case .alert: return .alert
-    }
+    } 
+  }
+  
+  public var excludedStops: [String] {
```
Comment (nighthawk) [inline Classes/model/CoreData/Alert.swift]: Please add a comment explaining what this is. What does it mean if an alert excludes stops? How is this typically used?

File: Classes/model/CoreData/Alert.h (hunk 1 above)
Comment (pnuts2) [reply Classes/model/CoreData/Alert.h]: Done

File: Classes/model/CoreData/Alert.swift (hunk 2 above)
Comment (pnuts2) [reply Classes/model/CoreData/Alert.swift]: Done

**Pull request merged**